examples/js-cloud-server/cloudflare-worker/README.md

```diff
@@ -0,0 +1,21 @@
+# DevCycle JS Cloud Bucketing Server SDK - Cloudflare Worker Example App
```

Comment:
whats the purpose of this example app? is it to provide an example for the public looking to use devcycle in a cloudflare worker? or is it a development aid for trying out the SDK the worker?

if its the former, I don't think it should be in the monorepo

if its the latter, it's not gonna be locally linked to the source files of the SDK like the other examples are

Reply:
As with all the example apps in this repo, its a bit of both.

Why do you say its not going to be linked?

Reply:
yea like it works as is right now, and the npm package isn't published yet.